src/components/InputBook.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { booksActions } from '../redux/books/books';
 
 function InputBook() {
+  const dispatch = useDispatch();
+  const handleAddBook = () => {
+    dispatch(booksActions.addBook({
+      id: 1,
+      title: 'Awa and Other plays',
+      author: 'Ndorrh Oswald',
+
+    }));
+  };
```

Comment:
Hi there. Normally you're supposed to dispatch an object with the data from the input fields and a new ID. Currently, you just add some predefined data with a constant id. So when we remove one the added books, all of them are deleted
 - [ ] Dispatch data from the input fields
 - [ ] Not two books are supposed to have the same ID so please kindly fix it